core/src/scss/utilities/variables/components/_main-nav.scss

```diff
@@ -0,0 +1,151 @@
+//        border--current:                    $color-driftwood,
+//        link--active-hover:                 $color-bright-red,
+//        border--active-hover:               $color-bright-red,
+//        link--expanded:                     $color-black,
+//        border--expanded:                   $color-driftwood,
+//        bg--mobile:                         $color-white,
+//        divider:                            #d9d9d9,
+//        link--mobile:                       $color-black,
+//        link--mobile-hover:                 $color-black,
+//        border--mobile-hover:               $color-black,
+//        link--mobile-active:                $color-bright-red,
+//        border--mobile-active:              $color-bright-red,
+//        link--mobile-expanded:              $color-black,
+//        border--mobile-expanded:            transparent,
+//        border--mobile-active-expanded:     $color-black,
+//    ),
+//    dark: (
+//        link:                               $color-white,
+//        link--hover:                        $color-white,
+//        border--hover:                      $color-white,
+//        link--active:                       $color-white,
+//        border--active:                     $color-vivid-red,
+//        link--active-hover:                 $color-white,
+//        border--active-hover:               $color-white,
+//        link--expanded:                     $color-white,
+//        link--expanded-hover:               $color-white,
+//        border--expanded:                   $color-white,
+//        border--expanded-hover:             $color-vivid-red,
+//        border--expanded-active:            $color-white,
+//    )
+//) !default;
+//
+// Styleguide: Variables.Components.main-nav-colors
+//
+$main-nav-colors: (
+  link:                                   $color-black,
+
+  link--hover:                            $color-bright-red,
+  border--hover:                          $color-bright-red,
+
+  link--active:                           $color-black,
+  border--active:                         $color-black,
+
+  link--active-hover:                     $color-bright-red,
+  border--active-hover:                   $color-bright-red,
+
+  link--expanded:                         $color-black,
+  border--expanded:                       $color-black,
+
+  bg--mobile:                             $color-black,
+  divider:                                $color-cool-grey,
+  link--mobile:                           $color-white,
+
+  link--mobile-hover:                     $color-white,
+  border--mobile-hover:                   $color-white,
+
+  link--mobile-active:                    $color-white,
+  border--mobile-active:                  $color-vivid-red,
+
+  link--mobile-expanded:                  $color-white,
+  border--mobile-expanded:                transparent,
+  border--mobile-active-expanded:         $color-white,
+) !default;
+
+$main-nav-light-colors: (
```

Comment:
These should have their own KSS comment.

Reply:
Added separate section with markup/comments for each variants